kythe/cxx/indexer/cxx/GraphObserver.h

```diff
@@ -974,8 +974,9 @@ class GraphObserver {
   /// \param Range The source range
   /// \param Signature The signature to use for the diagnostic VName
   /// \param Message The diagnostic message
-  virtual void recordDiagnostic(const Range& Range, std::string_view Signature,
-                                std::string_view Message) {}
+  virtual void recordDiagnostic(const Range& Range,
+                                const llvm::StringRef& Signature,
+                                const llvm::StringRef& Message) {}
```

Comment:
`llvm::StringRef` is like `absl::string_view` in that both should be passed by value rather than `const-&`

Reply:
Makes sense. I did it this way because I saw several other `StringRef&` parameters elsewhere in the file. I can send a followup to remove the `const &` from those.